variables.tf

```diff
@@ -327,6 +327,10 @@ variable "cloudfront" {
       allow_origins     = list(string)
       origin_override   = bool
     }))
+    response_header_to_remove = optional(object({
```

Comment:
Rather than take individual boolean options, I'd just use a list of strings:

```tf
remove_headers_config = optional(object({
    items = list(string)
}))
```

Reply:
done